docker-workshop/unitTest.rb

```diff
@@ -0,0 +1,89 @@
+require 'test/unit'
+require_relative 'testFramework'
+
+class TestBook < Test::Unit::TestCase
+    def test_create_obj
+        obj = Test_Framework.new
+        assert obj.instance_of? Test_Framework
+    end  
+
+    def test_navigate_to_url_success
+        obj = Test_Framework.new
+        # assert_equal("Navigated to url successfully", obj.navigate_to_url('C:\Users\Archita\Documents\Work_Infuse\Training\docker_selenium\test.html'))
+        assert_equal("Navigated to url successfully", obj.navigate_to_url('file:///app/test.html'))
+    end
+
+    def test_page_title
+        obj = Test_Framework.new
+        # obj.navigate_to_url('C:\Users\Archita\Documents\Work_Infuse\Training\docker_selenium\test.html')
+        obj.navigate_to_url('file:///app/test.html')
+        assert_equal("Mock Page for Unit Testing", obj.get_page_title)
+    end
+
+    def test_find_element
+        obj = Test_Framework.new
+        # obj.navigate_to_url('C:\Users\Archita\Documents\Work_Infuse\Training\docker_selenium\test.html')
+        obj.navigate_to_url('file:///app/test.html')
+        element = obj.find_element(xpath: "//*[@id='container']/div[2]/div[1]")
+        assert_equal("hello world", obj.get_text(element))
```

Comment:
Each unit test should test only one function. This unit test uses the `get_text` method of the framework.

Reply:
You can use the driver methods to get the text

Reply:
Change other unit tests similarly